src/homework21/person/PersonSort.java

```diff
@@ -0,0 +1,120 @@
+package homework21.person;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Scanner;
+
+public class PersonSort {
+    private String name;
+    private String surname;
+    private String gender;
+    private Integer age;
+
+    public PersonSort(String name, String surname, String gender, Integer age) {
+        this.name = name;
+        this.surname = surname;
+        this.gender = gender;
+        this.age = age;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getSurname() {
+        return surname;
+    }
+
+    public String getGender() {
+        return gender;
+    }
+
+    public Integer getAge() {
+        return age;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public void setSurname(String surname) {
+        this.surname = surname;
+    }
+
+    public void setGender(String gender) {
+        this.gender = gender;
+    }
+
+    public void setAge(Integer age) {
+        this.age = age;
+    }
+
+    static List<PersonSort> list = new ArrayList<>();
```

Comment:
This list is unexpected here, because each method should work independently (it means that we should pass List to each needed method, instead of using general one).
Don't forget to ask me to clarify on the lesson about it